clearwater_config_manager/default_ifcs_config_plugin.py

```diff
@@ -0,0 +1,77 @@
+# Project Clearwater - IMS in the Cloud
+# Copyright (C) 2017  Metaswitch Networks Ltd
```

Comment:
Switch out for new copyright statement
(and same for all other new files in this pull request)